experimental/veggieseasons/lib/screens/search.dart

```diff
@@ -47,11 +47,14 @@ class _SearchScreenState extends State<SearchScreen> with RestorationMixin {
   }
 
   Widget _createSearchBox() {
+    final themeData = CupertinoTheme.of(context);
     return Padding(
       padding: const EdgeInsets.all(8),
-      child: SearchBar(
+      child: CupertinoSearchTextField(
+        backgroundColor: Styles.searchBackground(themeData),
```

Comment:
Now that there's an official component, let's go with the default background color and style. It wasn't an option before, but it's a great chance to show off how CupertinoSearchTextField works.

I'd cut lines 54 and 57, and remove anything in `lib/styles.dart` that's no longer used (probably the two things you would have just cut references to 😄).

Reply:
Thank you @RedBrogdon  for reviewing my PR  I'll do the required changes 😊.